contracts/Controller.sol

```diff
@@ -47,25 +51,25 @@ contract Controller is Ownable {
     /**
      * @notice modifier check if protocol is not paused
      */
-    // modifier isNotPaused {
-    //     _;
-    // }
+    modifier isNotPaused {
+        require(!systemPaused, "Controller: system is paused");
 
-    /**
-     * @notice modifier to check if otoken is expired
-     * @param _otoken otoken address
-     */
-    // modifier isExpired(address _otoken) {
-    //     _;
-    // }
+        _;
+    }
 
     /**
-     * @notice modifier to check if sender is an authorized vault operator
+     * @notice modifier to check if sender is an account owner or an authorized account operator
      * @param _sender sender address
+     * @param _accountOwner account owner address
      */
-    // modifier isAuthorized(address _sender) {
-    //     _;
-    // }
+    modifier isAuthorized(address _sender, address _accountOwner) {
```

Comment:
why not just have this be msg.sender? Does this need to take in _sender as an input?

Reply:
nope, thought just for more clarity.